python/dgl/graphbolt/dataloader.py

```diff
@@ -61,6 +66,44 @@ def __iter__(self):
         yield from self.dataloader
 
 
+class ThreadingWrapper(dp.iter.IterDataPipe):
```

Comment:
can we utilize https://pytorch.org/data/main/generated/torchdata.datapipes.iter.ThreadPoolMapper.html to achieve this?